tests/platformtests/test_security_tag.py

```diff
@@ -128,6 +128,72 @@ def test_securitytag(self):
 
         self.assertTrue(resource_id is None)
 
+    def test_Yet_Another_securitytag(self):
+        """Deploying security tag minimal test"""
+
+        # set blueprint name
+        blueprint = os.path.join(
+            self.blueprints_path,
+            'security_tag/blueprint.yaml'
+        )
+
+        #set inputs name
+        inputs = os.path.join(
```

Comment:
We will never have credentials in input files for tests, so use os.environ.get('*') for now.